scrapers/MindGeek.yml

```diff
@@ -58,6 +59,7 @@ performerByURL:
       - lookathernow.com/model/
       - mofos.com/model/
       - mofosnetwork.com/model/
+      - realitykings.com/scene/
```

Comment:
it should be 
```yaml
      - realitykings.com/model/
```
but double check that the layout is the same. I think it needs some minor adjustments to work correctly